src/EditorFeatures/VisualBasic/SplitComment/VisualBasicSplitCommentService.vb

```diff
@@ -0,0 +1,47 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.Editor.Implementation.SplitComment
+Imports Microsoft.CodeAnalysis.Host.Mef
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.SplitComment
+    <ExportLanguageService(GetType(ISplitCommentService), LanguageNames.VisualBasic), [Shared]>
+    Partial Friend Class VisualBasicSplitCommentService
+        Implements ISplitCommentService
+
+        <ImportingConstructor>
+        <Obsolete(MefConstruction.ImportingConstructorMessage, True)>
+        Public Sub New()
+        End Sub
+
+        Public ReadOnly Property CommentStart As String Implements ISplitCommentService.CommentStart
+            Get
+                Return "'"
+            End Get
+        End Property
+
+        Public Function IsAllowed(root As SyntaxNode, trivia As SyntaxTrivia) As Boolean Implements ISplitCommentService.IsAllowed
+            ' we don't allow splitting the comment if there is a preceding line continuation character.  This is
+            ' primarily because this is not a common enough scenario to warrant the extra complexity in this fixer.
+
+            Dim currentTrivia = trivia
+            While currentTrivia <> Nothing AndAlso currentTrivia.SpanStart > 0
+                Dim previousTrivia = root.FindTrivia(currentTrivia.SpanStart - 1)
+                If previousTrivia.Kind() = SyntaxKind.LineContinuationTrivia Then
```

Comment:
```suggestion
                If previousTrivia.IsKind(SyntaxKind.LineContinuationTrivia) Then
```

Reply:
i find these equal :)  out of curiosity, do you have an argument for one or the other?

Reply:
`IsKind` doesn't need to do a range check.

https://github.com/dotnet/roslyn/blob/afd10305a37c0ffb2cfb2c2d8446154c68cfa87a/src/Compilers/CSharp/Portable/CSharpExtensions.cs#L25-L27

https://github.com/dotnet/roslyn/blob/afd10305a37c0ffb2cfb2c2d8446154c68cfa87a/src/Compilers/CSharp/Portable/CSharpExtensions.cs#L188

Reply:
good enough for me :)